src/lib/validations/sql.js

```diff
@@ -353,15 +363,15 @@ export const staticSqlValidations = {
 };
 
 // For standalone SQL validations
-export const validate = async ( filename: string, isImport: boolean = false ) => {
+export const validate = async ( filename: string, skipChecks: string[] = [ 'useStatement' ] ) => {
```

Comment:
`isImport` was never used so I removed it.

I added new optional parameter to skip checks... and default to skip this new useStatement validation. From dev-env we send an empty array so that all validation goes.